libgnucash/core-utils/core-utils.i

```diff
@@ -59,6 +59,9 @@ gchar * gnc_path_get_bindir(void);
 %newobject gnc_path_get_localedir;
 gchar * gnc_path_get_localedir(void);
 
+/* Name of our gettext-domain (defined in config.h) */
```

Comment:
JFYI: it is set in https://github.com/Gnucash/gnucash/blob/maint/CMakeLists.txt#L32 and then published via config.h

Reply:
I saw that but I think that's specific enough for this comment.